serverless.yml

```diff
@@ -180,6 +182,28 @@ resources:
           - AttributeName: 'updatedAt'
             AttributeType: 'N'
 
+    tokenInfoTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: ${self:custom.table.TOKEN_INFORMATION_DATA}
+        BillingMode: PAY_PER_REQUEST
+        KeySchema:
+          - AttributeName: 'address'
+            KeyType: 'HASH'
+          - AttributeName: 'updatedAt'
+            KeyType: 'RANGE'
+        AttributeDefinitions:
+          - AttributeName: 'address'
+            AttributeType: 'S'
+          - AttributeName: 'name'
+            AttributeType: 'S'
+          - AttributeName: 'symbol'
```

Comment:
ddb will probably complain about this try running `yarn dev` to see if the migration works.
you will probably need to remove all definitions outside of `address` and `updatedAt`